msvc/sdk-build.props

```diff
@@ -35,6 +35,7 @@
     <WindowsTargetPlatformMinVersion Condition="'$(WindowsTargetPlatformMinVersion)'==''">10.0.10586.0</WindowsTargetPlatformMinVersion>
     <EnableDotNetNativeCompatibleProfile Condition="'$(EnableDotNetNativeCompatibleProfile)'==''">true</EnableDotNetNativeCompatibleProfile>
     <IncludeOutputsInPackage>false</IncludeOutputsInPackage>
+    <IncludeFrameworkReferencesInPackage>false</IncludeFrameworkReferencesInPackage>
```

Comment:
This causes GetPackageContents to not depend on resolve references. See [here](https://github.com/NuGet/NuGet.Build.Packaging/blob/2fa6c67fd099ba08855c930de727f195d4ed1704/src/Build/NuGet.Build.Packaging.Tasks/NuGet.Build.Packaging.targets#L199-L201)